### ISSUE TITLE
rowspan and page break

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#361]
```
What steps will reproduce the problem?
1. a table cell that has rowsapn > 1
2. page break that is calculated to a row that "splitts" that cell
3. That should do it...

What is the expected output? What do you see instead?
Rendering fails

What version of the product are you using? On what operating system?
DOMPDF 0.6.0 beta2

Please provide any additional information below.

I solved it by inserting:
1. in Table_Row_Group_Frame_Decorator::split()
after:     $cellmap = $this->get_parent()->get_cellmap();

    while($child->get_node()->getAttribute("dontbreak")){
        $child = $child->get_prev_sibling();
    }

2. in the html <tr "dontbreak"="1"> for each row that has cells that starts 
before it.

Thank you for that wonderful software.
Yariv
ydb1976@gmail.com
```

Original issue reported on code.google.com by `ydb1...@gmail.com` on 11 Oct 2011 at 3:30
- Merged into: #61

**Comment by GoogleCodeExporter** [comment]
```
We've started work on a fix for this problem, plus we hope to have further 
improvement to the table-handling code for the final 0.6.0 release. Your patch 
may help in the meantime for anyone experiencing the issue.
```

Original comment by `eclecticgeek` on 12 Oct 2011 at 2:13
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
About my patch: WARNING!  it might lead to infinite loop of trying to place too 
high "cluster" of successive "dontbreak" rows.
```

Original comment by `ydb1...@gmail.com` on 12 Oct 2011 at 3:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:16
- Added labels: **Restrict-AddIssueComment-Commit**
